tests/functional/firefox/test_iostestflight.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Yeah best update that one too. Current convention is to separate via underscores in test file names for readability 👍